### ISSUE TITLE
Create a separate bundle for each environment

### ISSUE THREAD

**Issue by liketurbo** [toncenter/tonweb#78]
I have [Bun](https://bun.sh) project, when I've tried to use this library in my project I've got an error saying: "navigator is not defined...", which means that I'm using a web browser version.

Upon further investigation I noticed that code used from the library located in the `./src` folder, not in the `./dist` (specified in the `package.json`'s field - `"main": "src/index.js"`). Which raises the question - how this code even works in a browsers if they're using CommonJS module system 🤔. @tolya-yanot Would you carry to explain how it works, cause I'm obviously something missing 😅

In summary `./dist/tonweb.js` - works for a browser, but not for node, `./src/index.js` - works for node, but not for a browser. And neither one of them works for [Bun](https://bun.sh).

My proposal is to create a separate js bundle for each environment (`tonweb/lib-node`, `tonweb/lib-web`) to get rid of this ambiguity and make the library compatible with [Bun](https://bun.sh).  

\* _Bun - modern JavaScript runtime like Node or Deno_
